Automattic/wp-calypso
# Purchases: Refactor Confirm Cancel Purchase to use loading placeholder

Part of #275.

This is how loading placeholder should look like for this page:
![screen shot 2015-10-21 at 1 44 37 pm](https://cloud.githubusercontent.com/assets/12596797/10645211/95cfb784-77fa-11e5-9626-e6e799e40f12.png)

### Testing
1. Go to http://calypso.localhost:3000/purchases.
2. Navigate to purchase within refund period.
3. Navigate to `Cancel Purchase` page using `Cancel and Refund` link.
4. Refresh page and make sure loading placeholder still works properly.
5. Click on `Cancel and Refund` button.
6. Make sure loading placeholder looks like on the screen above.
7. Refresh page and click back button in header cake when page is loading and make sure it works properly and there are no errors in JS console.
8. Click on `Cancel and Refund` button.
9. Click back button in header cake after page is fully loaded and make sure it loads the same page as in (7).
10. Click on `Cancel and Refund` button.
11. Make sure it's possible to cancel your purchase.

### Review
